<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessTravelers</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabGame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QPushButton" name="SelectGame">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Statistic">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistic</string>
      </property>
     </widget>
     <widget class="QPushButton" name="About">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>180</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ChessTravelers&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>SelectGame</string>
     </attribute>
     <widget class="QPushButton" name="newGame">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Load">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackOutSelect">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="GameLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Game</string>
     </attribute>
     <widget class="QPushButton" name="Menu">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>1</y>
        <width>405</width>
        <height>405</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="playerName_label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>101</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>124</red>
            <green>101</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>50</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QPushButton" name="BackOutStatistic">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableBestPlayers">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>501</width>
        <height>361</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Best players</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QPushButton" name="BackOutAbout">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TextAbout">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>0</y>
        <width>521</width>
        <height>381</height>
       </rect>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>EnterName</string>
     </attribute>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="EnterNameLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter your name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backSelect">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NamelineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>411</width>
        <height>91</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>6</red>
            <green>106</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>6</red>
            <green>106</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_ver">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>GameVersion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
